lighthouse-core/lib/emulation.js

```diff
@@ -86,7 +86,7 @@ function enableNexus5X(driver) {
     // Network.enable must be called for UA overriding to work
     driver.sendCommand('Network.enable'),
     driver.sendCommand('Network.setUserAgentOverride', NEXUS5X_USERAGENT),
-    driver.sendCommand('Emulation.setTouchEmulationEnabled', {
+    driver.sendCommand('Page.setTouchEmulationEnabled', {
```

Comment:
first real bug caught with type checks? Haven't confirmed that it was a problem, but there is both [`Emulation.setTouchEmulationEnabled`](https://chromedevtools.github.io/devtools-protocol/tot/Emulation#method-setTouchEmulationEnabled) and [`Page.setTouchEmulationEnabled`](https://chromedevtools.github.io/devtools-protocol/tot/Page#method-setTouchEmulationEnabled), but only the `Page` one takes these params

Reply:
ha! yeah I think this might have been a small bug that normally didn't matter.
I think we want [`Emulation.setEmitTouchEventsForMouse`](https://chromedevtools.github.io/devtools-protocol/tot/Emulation#method-setEmitTouchEventsForMouse) instead.

that's what devtools uses when you toggle device type:
![image](https://user-images.githubusercontent.com/39191/38268215-5f145304-3732-11e8-9fe7-d0c98c693a13.png)

Reply:
seems like you're fixing this in a separate PR?

can you write up the list of changes that are on their way?

Reply:
sorry, I can fix this :)